### PR TITLE
Update translations using inlang

### PR DESCRIPTION
**Pull request by mbaraa**
INLANG

Add missing Arabic [ar-SA] translations, and fixed some inaccurate translations.

### CODE REVIEW

**Review by richardshiue — APPROVED** [review]

**Comment by richardshiue** [comment]
Hey there @mbaraa, thanks for contributing. Please merge upstream into your branch and resolve the conflicts

**Comment by mbaraa** [comment]
hey @richardshiue, I resolved the conflicts with upstream.

File: frontend/resources/translations/en.json
Comment (mbaraa) [reply frontend/resources/translations/en.json:986]: yeah sure.

Comment (mbaraa) [reply frontend/resources/translations/en.json]: done.

**Review by richardshiue — APPROVED** [review]
LGTM, I'll merge after the tests pass

**Comment by mbaraa** [comment]
Is the commit's prefix a big deal? 👀

**Comment by richardshiue** [comment]
> Is the commit message's prefix a big deal? 👀

No, not really. Maintainers can edit the commit message while merging the PR. But some don't like to see the red x lol

**Comment by mbaraa** [comment]
> > Is the commit message's prefix a big deal? 👀
> 
> No, not really. Maintainers can edit the commit message while merging the PR. But some don't like to see the red x lol

fair enough.

**Pull request merged**
